### PR TITLE
add translation it-it, pt-br for developers_dev.md

### PR DESCRIPTION
**Pull request by jeremy0519**
#250

### CODE REVIEW

**Comment by jeremy0519** [comment]
Please ignore the first 3 commits.

**Comment by jeremy0519** [comment]
#251

**Comment by jeremy0519** [comment]
#264

**Comment by jeremy0519** [comment]
github仓库名（wuhan2020/2019-nCov-Nav）和slack channel名（#anti-2019-ncov）没有更改

**Review by GindaChen — APPROVED** [review]
LGTM! Thanks!

**Comment by chenrui333** [comment]
relates to https://github.com/wuhan2020/wuhan2020/pull/369

File: i18n/en-us.js [hunk 1]
```diff
@@ -209,21 +209,21 @@ export default {
...
           repoLink: 'https://github.com/wuhan2020/wuhan2020.github.io',
-          description: 'Wuhan2020 community calls for volunteers all over the world to establish an open-source information platform and contribute to the fight against the 2019-nCoV',
+          description: 'Wuhan2020 community calls for volunteers all over the world to establish an open-source information platform and contribute to the fight against the COVID-19',
           link: 'https://community.wuhan2020.org.cn/',
         },
         {
           img: '/images/blog/wuhan2020.png',
-          title: '2019-nCoV Outbreak Navigation Site',
+          title: 'COVID-19 Outbreak Navigation Site',
           repoLink: 'https://github.com/wuhan2020/2019-nCoV-Nav',
```
Comment (chenrui333) [inline i18n/en-us.js:225]: Now we can update the repoLink as well.

**Comment by jeremy0519** [comment]
I wonder why there are 3 older commits submitted together with the latest commit.

**Comment by chenrui333** [comment]
Ideally, you should have a good git workflow, I wrote a pretty good tutorial in here about how to do it right.

**Comment by chenrui333** [comment]
https://github.com/k8smeetup/k8s-official-translation/blob/master/branching-strategy.md

**Comment by jeremy0519** [comment]
Thanks a lot.

**Comment by jeremy0519** [comment]
So I need to checkout a new branch, right?

**Comment by chenrui333** [comment]
yeah, every time.

**Comment by chenrui333** [comment]
Since currently, your dev branch is quite polluted, I would suggest that you should delete and forced pushed to update your fork (if you want), and use the new upstream dev branch as your base branch every single time.

**Comment by chenrui333** [comment]
Let me know if that makes sense.

**Comment by jeremy0519** [comment]
I have added you as a collaborator, can you help me do this?
I don't quite understand how to use the new upstream dev branch as my base branch.

**Comment by chenrui333** [comment]
Basically it would be like this
- git checkout master
- git branch -D dev
- git checkout -b dev upstream/dev
- git push -f origin dev
- git checkout -b new-work-branch # new work

**Comment by chenrui333** [comment]
```
$ git push -f jeremy0519 dev
Total 0 (delta 0), reused 0 (delta 0)
To github.com:jeremy0519/wuhan2020.github.io.git
 ! [remote rejected] dev -> dev (permission denied)
error: failed to push some refs to 'git@github.com:jeremy0519/wuhan2020.github.io.git'
```

**Comment by jeremy0519** [comment]
I think you should accept the collaborator invitation somewhere in your account.

**Comment by chenrui333** [comment]
Following the above instructions, you should be fine. I going to sleep now. G'night.

**Comment by jeremy0519** [comment]
I've got it all right now, thanks.

**Pull request merged**